CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v7.55.0
+------------------------------
+*November 21, 2023*
+
+### Changed
+- Removes support for deprecated Node versions (12 and 14).
```

Comment:
This update is awesome!

I think we'll likely need to keep `v14` in the engines as I know some legacy Customer platforms use this (ones that are being deprecated by OneWeb and so won't be updating to latest versions of Node, like coreweb for instance)

Reply:
And also just realised some legacy projects running on legacy build agents don't have access to Node >=16! Shame
Will revert back v14!